.gitignore

```diff
@@ -8,5 +8,41 @@ terraform/modules/kube-base-components/testapp/node_modules
 **/go.mod
 **/go.sum
 **/node_modules
-terraform.tfstate**
-# kube-prometheus-stack-**.tgz
\ No newline at end of file
+
+# Terraform files
+**/terraform.tfvars
+**/.terraform
+**/.terraspace-cache
+**/*.tfstate
+**/*.tfstate.backup
+**/*.terraform.lock.hcl
+**/*.terraform.tfstate.lock.info
+**/*.plan
+
+# OS X files
+.history
+.DS_Store
+
+# Common ruby files
+*.gem
+*.rbc
+.env*
+.idea
+/.bundle
+/.config
+/.yardoc
+/_yardoc
+/coverage
+/doc/
+/Gemfile.lock
+/InstalledFiles
+/lib/bundler/man
+/log
+/output
+/pkg
+/rdoc
+/spec/fixtures/project/output
+/spec/reports
+/test/tmp
+/test/version_tmp
+/tmp
```

Comment:
I'm not sure you need this in the root, it makes more sense to be in `terraspace/.gitignore` no?

Reply:
Well most projects that I see usually have .gitignore in root.. Maybe placing it anywhere else might be confusing.
A newcomer might take a look at root gitignore and think that stuff is missing (Because this person didn't see that there was a second .gitignore)

Reply:
Yeah I'd keep it in root...